lending-pool/src/auctions/bad_debt_auction.rs

```diff
@@ -82,6 +82,13 @@ pub fn fill_bad_debt_auction(
         &filler_state.address,
     );
 
+    // If the backstop still has liabilities and less than 10% of the backstop threshold burn bad debt
+    if backstop_state.positions.liabilities.len() > 0
+            //TODO: this token check needs to check k-value of pool balance LP tokens
+        && backstop_client.pool_balance(&e.current_contract_address()).tokens < 100_000_0000
```

Comment:
does 100k represent the kValue or representative Blend Tokens? I'm gonna assume the former

Reply:
Think the underscores confused me, this is 100 tokens at the moment?

Do we have the k-value threshold we are going to use for the pool balance figured out?

Reply:
yeah threshold is 200,000 based on the fact that it costs around 10k to spin up a pool at 1.2 cent blend and 50k at 10 cent blend

I'll update the amounts to reflect that